### DISCUSSION TITLE
What is CLI command for many to many relationship?

### DISCUSSION THREAD

**Discussion by onesquarepeg** [blitz-js/blitz#3139] [Q&A]
1. What is the right way to implement a many to many relationship with all the boilerplate code?

2. How do I rename my table after having created it?

**Comment by beerose** [comment]
1. By boilerplate do you mean pages/forms etc? Currently, we only generate a skeleton of these.

2. You can go to your `db/schema.prisma` file, rename the table, and then run `blitz prisma migrate dev` to generate a new migration and apply changes to the database.
